### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by gracerx** [novemberfiveco/symbol-spacer-sketch-plugin#11]
At a client job I recently got another laptop and instance of Sketch to work. I downloaded and installed the SymbolSpacer plugin. It doesn't show up in the plugins menu, nor does it appear in the plugins directory. I tried copying the plugin to the directory but that didn't work either. 

Today I opened my primary laptop with sketch on it, and opened the sketch plugin manager, searched for Symbol Spacer, clicked on the update button, and now SymbolSpacer doesn't appear/work. The plugin is in plugins directory. 

Primary Mac: Sketch Version 47.1/(45422), MBP: macOS High Sierra, Version 10.13.1
Secondary Mac: Sketch Version 47.1/(45422), MBP: macOS Sierra, Version 10.12.6

**Comment by seansmyth** [comment]
Yeah, this doesn't work for me. Just downloaded it today but sadly, no joy :(

**Comment by TuiMediaSam** [comment]
@GeertWille @gracerx @seansmyth 

I found an intermittent fix for the installation issue - posting here just in case it helps others, or the creator to fix the issue - I use this plugin daily and would love to be able to reliably install it so that my team could use it in our workflow.

**My situation / view of the bug:**
Installing on some machines flat out refuses…no error message, plugin DOES appear in the sketch plugins folder, but DOES NOT function or appear in the Sketch plugins menu. The affected (non-installing) machines are a mix of Sierra and High Sierra (DOES work on my High Sierra v10.13.2 although this may be subject to the detail in the solution I found, DOES NOT work on my Sierra v10.12.6, or my Sierra 10.13.2 - all with Sketch 48.2).

**The fix I found:**
I managed to get the plugin to install correctly by… 

1. Download an older version of Sketch - specifically Sketch v45 (43475) - from here: https://www.sketchapp.com/updates/.
2. Clear previous unsuccessful installations of the symbol spacer plugin by going Sketch > Plugins > Manage plugins > Show plugins folder, then trashing any instance of the symbol spacer plugin, empty trash.
3. Close any open documents and quit Sketch completely.
4. Open the newly downloaded Sketch v45, download a fresh copy of the symbol spacer plugin from the github source zip, then install it in Sketch v45 by either dragging it onto Sketch v45, or CTRL+Click on it > Get Info > Set 'Open with:' to Sketch 45), then double-click the plugin file as usual. 

You will either see the 'Plugin installed successfully' message, after which you can delete Sketch v45, re-open the latest version of Sketch (v48.2 at time of writing) and voila! - plugin working, or you'll see nothing at all and you're back to square one. 

Note: I've found it to be successful only 50% of the time, and discovered it by trial and error so the underlying cause of the issue I do not know, nor why this fix (sometimes) works. Also worth mentioning that even if the plugin DOES NOT install, it will put a copy of itself into the Sketch plugins folder, it's probably best to remove this.

I really hope this helps, I have the plugin working on some machines but not others and it's driving me crazy because it is fantastically useful.
